app.py

```diff
@@ -49,13 +49,14 @@ def main_prediction():
     """
     st.markdown(html_temp,unsafe_allow_html=True)
     st.markdown("""<br>""",unsafe_allow_html=True)
-    x = st.text_input("X")
-    y = st.text_input("Y")
-    month = st.text_input("Month")
-    ffmc = st.text_input("FFMC")
-    dmc = st.text_input("DMC")
-    dc = st.text_input("DC")
-    temp = st.text_input("Temp")
+    x = st.text_input("X","Enter an integer e.g. 5")
+    y = st.text_input("Y","Enter an integer e.g. 5")
+    month = st.text_input("Month","Enter a numeric representation of the month e.g. if February enter 2")
```

Comment:
`for` February enter 2

Reply:
You should be saying that when I asked you checked those screenshots. Just make it at once.
Every time I am pushing the code I am asking you to check each and every feature I am adding. And you were saying yes its ok. Now why you are turning now, the link of the deployed app is always the same.